assignments/prototypes.js

```diff
@@ -32,7 +49,17 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+ function Humanoid(human){
+   CharacterStats.call(this, human);
+   this.team = human.team;
+   this.weapons = human.weapons;
+   this.language = human.language;
+ }
+ Humanoid.prototype = Object.create(CharacterStats.prototype);
+ Humanoid.prototype.greet = function(greeting){
```

Comment:
Why do you have a greeting parameter but never called it? Maybe it could have just been left out?